server/src/routes.ts

```
@@ -475,4 +475,40 @@ export default async function routes(fastify: FastifyInstance) {
       return;
     }
   );
+
+  //********************************************************************
+  // TRANSACTIONS
+  //********************************************************************
+
+  fastify.get(
+    '/transactions',
+    {
+      // preValidation: [fastify.authenticate],
+      schema: {
+        querystring: {
+          limit: { type: 'number' },
+          offset: { type: 'number' },
+        },
+      }
+    },
+    async (req, res) => {
+      const limit = parseInt(req.query.limit) || 0
+      const offset = parseInt(req.query.offset) || 0
+
+      const transactions = await getTransactions(limit, offset);
+      const totalTransactions = await getTotalTransactions();
+
+      if (!transactions) {
+        return new createError.NotFound('Invalid Event');
```

Please, let's raise an error that makes sense